lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
Is this going to work for both /post and /collection?
I think the correct thing is just to always do `vm.view = response.data` , whether it's an array or single object.
We need to fix the json-server so it sends a single obj instead of an array for /post only (issue already in ng-web-app).